package.json

```diff
@@ -119,14 +116,6 @@
         "./examples/typescript:build"
       ]
     },
-    "types:check": {
```

Comment:
Our contributing guide still has `pnpm run types:check` from root as something you should do. Also, it looks like `types:check` are still available and runnable in our other packages / examples?

Reply:
Yeah I can align it to run the same `tsc` command as our other packages. Good call out. `tsup` also does type checking as part of it's process so if you have invalid types, the thing will fail to build.